pkg/actions/tools/tox/tox.go

```diff
@@ -0,0 +1,24 @@
+// package tox contains tox related actions
+package tox
+
+import (
+	"strings"
+
+	"github.com/carapace-sh/carapace"
+)
+
+// ActionEnvironments completes tox environments
+func ActionEnvironments() carapace.Action {
+	return carapace.ActionCallback(func(c carapace.Context) carapace.Action {
+		cmd := c.Command("tox", "--listenvs-all")
```

Comment:
This is fine as it uses `c.Command` which handles context correctly (e.g. setting environment variables).
But there's also [ActionExecCommandE](https://carapace-sh.github.io/carapace/carapace/defaultActions/actionExecCommandE.html) which is intended for custom error handling and might be an option here.

Reply:
On second thought [ActionExecCommand](https://carapace-sh.github.io/carapace/carapace/defaultActions/actionExecCommand.html) might be sufficient here.